core/runtime/src/main/java/io/quarkus/runtime/logging/WindowsConsoleDirectives.java

```diff
@@ -0,0 +1,38 @@
+package io.quarkus.runtime.logging;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+
+import org.graalvm.nativeimage.Platform;
+import org.graalvm.nativeimage.Platforms;
+import org.graalvm.nativeimage.c.CContext;
+
+import com.oracle.svm.core.util.VMError;
+
+@Platforms(Platform.WINDOWS.class)
+public class WindowsConsoleDirectives implements CContext.Directives {
+
+    private static final String[] windowsLibs = new String[] { "<windows.h>" };
```

Comment:
currently I'm getting an error I highly assumes is missing header files but not sure.
Error is:
```
Error: Error compiling query code (in C:\Users\max\AppData\Local\Temp\SVM-3772467201033451472\NativeInfoDirectives.cpp). Compiler command  CL C:\Users\max\AppData\Local\Temp\SVM-3772467201033451472\NativeInfoDirectives.cpp /FeC:\Users\max\AppData\Local\Temp\SVM-3772467201033451472\NativeInfoDirectives.exe output included error: [Microsoft (R) C/C++ Optimizing Compiler Version 16.00.30319.01 for x64, Copyright (C) Microsoft Corporation.  All rights reserved., ]
com.oracle.svm.core.util.UserError$UserException: Error compiling query code (in C:\Users\max\AppData\Local\Temp\SVM-3772467201033451472\NativeInfoDirectives.cpp). Compiler command  CL C:\Users\max\AppData\Local\Temp\SVM-3772467201033451472\NativeInfoDirectives.cpp /FeC:\Users\max\AppData\Local\Temp\SVM-3772467201033451472\NativeInfoDirectives.exe output included error: [Microsoft (R) C/C++ Optimizing Compiler Version 16.00.30319.01 for x64, Copyright (C) Microsoft Corporation.  All rights reserved., ]
        at com.oracle.svm.core.util.UserError.abort(UserError.java:114)
        at com.oracle.svm.hosted.c.NativeLibraries.reportErrors(NativeLibraries.java:196)
        at com.oracle.svm.hosted.NativeImageGenerator.processNativeLibraryImports(NativeImageGenerator.java:1512)
        at com.oracle.svm.hosted.NativeImageGenerator.setupNativeLibraries(NativeImageGenerator.java:993)
        at com.oracle.svm.hosted.NativeImageGenerator.setupNativeImage(NativeImageGenerator.java:827)
        at com.oracle.svm.hosted.NativeImageGenerator.doRun(NativeImageGenerator.java:520)
        at com.oracle.svm.hosted.NativeImageGenerator.lambda$run$0(NativeImageGenerator.java:440)
        at java.util.concurrent.ForkJoinTask$AdaptedRunnableAction.exec(ForkJoinTask.java:1386)
        at java.util.concurrent.ForkJoinTask.doExec(ForkJoinTask.java:289)
        at java.util.concurrent.ForkJoinPool$WorkQueue.runTask(ForkJoinPool.java:1056)
        at java.util.concurrent.ForkJoinPool.runWorker(ForkJoinPool.java:1692)
        at java.util.concurrent.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:157)
Error: Image build request failed with exit status 1
```

waiting for https://github.com/oracle/graal/issues/1675 to be fixed or someone magically figuring out what is wrong ;)

Reply:
Can you show the content of `C:\Users\max\AppData\Local\Temp\SVM-3772467201033451472\NativeInfoDirectives.cpp`?